packages/commerce-sdk-react/src/hooks/ShopperCustomers/cache.ts

```diff
@@ -84,7 +79,13 @@ export const cacheUpdateMatrix: CacheUpdateMatrix<Client> = {
             remove: [{queryKey: getCustomerPaymentInstrument.queryKey(parameters)}]
         }
     },
-    deleteCustomerProductList: TODO('deleteCustomerProductList'),
+    deleteCustomerProductList(customerId, {parameters}) {
+        return {
+            // TODO: Rather than invalidate, can we selectively update?
+            invalidate: [{queryKey: getCustomerProductLists.queryKey(parameters)}],
+            remove: [{queryKey: getCustomerProductList.queryKey(parameters)}]
```

Comment:
I found when removing queries from cache the fuzzy matching with the `path` works best, since we want to remove getCustomerProductLists, getCustomerProductList, getCustomerProductListItem, getPublicProductListsBySearchTerm etc.
```suggestion
            remove: [{queryKey: getCustomerProductList.path(parameters)}]
```